ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -317,6 +327,36 @@ imported payload, the semaphore's prior permanent payload will be restored.
 Please refer to handle specific specifications for more details on transference and
 permanence requirements specific to handle type.
 
+To import a handle of type _handle_type_ into an existing semaphore, call the function
+
+include::{generated}/api/protos/clImportSemaphoreKHR.txt[]
+
+_sema_object_ specifies a valid semaphore object with importable properties.
```

Comment:
Do we strictly need to pass handle value at the time of import if we can associate it with this new API at a later point?
We don't need to disallow, but would it make sense to make it optional or allow the handle value to be NULL?